footer-component.js

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Footer Component.
+ *
+ * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.component('brFooter', {
+    controller: Ctrl,
+    templateUrl: requirejs.toUrl(
+      'bedrock-angular-footer/footer-component.html'),
+  });
+}
+
+/* @ngInject */
+function Ctrl($rootScope, config) {
+  var self = this;
+  self.route = $rootScope.route;
```

Comment:
I don't think this works, as the `br-footer` component is presently shown outside of `ng-view` in `bedrock-angular`. Thoughts?